lib/facter/jenkins.rb

```diff
@@ -12,3 +12,24 @@
     plugins.keys.sort.map { |plugin| "#{plugin} #{plugins[plugin][:plugin_version]}" }.join(', ')
   end
 end
+
+Facter.add(:jenkins_version) do
+  confine kernel: 'Linux'
+
+  setcode do
+    war = case Facter.value('os')['family']
```

Comment:
Doesn't this also work?
```suggestion
    war = case Facter.value('os.family')
```